### ISSUE TITLE
OBJ file exporter

### ISSUE THREAD

**Issue by aylakhan** [SCIInstitute/SCIRun#1239]
Port [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file) file exporter and fix parsing problems (doesn't fully support spec).

**Comment by dcwhite** [comment]
@aylakhan Will the ofstream fix close this one? How do I test it?

**Comment by aylakhan** [comment]
No, it won't. Basically, the exporter does not support the full OBJ spec. If we were to carefully document the fact that the exporter produces files that are probably only readable by ImageVis3D (and likely not readable by any other software that reads OBJ files), we could close it.

**Comment by dcwhite** [comment]
@jessdtate What's the priority of this one?

**Comment by jessdtate** [comment]
no one is clamoring for it right now.  so not so high

**Comment by dcwhite** [comment]
**Triage 2026-06:** OBJ file exporter — porting from v4 with parser improvements. Low priority per the last comment. An OBJ exporter would output triangle mesh geometry from TriSurf fields. The implementation would be a new class in `src/Core/Algorithms/DataIO/` following the existing field exporter pattern. Medium effort when prioritized.
